.circleci/config.yml

```diff
@@ -1,48 +1,121 @@
 version: 2.1
 
+executors:
+  python:
+    parameters:
+      version:
+        type: string
+        default: "3.7"
+
+    docker:
+      - image: circleci/python:<< parameters.version >>
+
+commands:
+  installation:
+    parameters:
+      test_requirements:
+        type: boolean
+        default: false
+
+    steps:
+      - checkout
+      - run:
+          name: "Create virtual environment and install requirements."
+          command: |
+            virtualenv python -q
```

Comment:
I think naming this folder python is confusing given that python is also a command